src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4680,4 +4680,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning" xml:space="preserve">
+    <value>Attribute '{0}' is ignored when public signing is specified.</value>
+  </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning_Title" xml:space="preserve">
```

Comment:
I originally did not include it, but this test failed and so I threw it in. https://github.com/dotnet/roslyn/blob/fd32a7348150294a5f9d3a212da67d41977b2f12/src/Compilers/CSharp/Test/Semantic/Diagnostics/DiagnosticAnalyzerTests.cs#L719